.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+version: '3.4'
+
+services:
+
+  rabbitmq:
+    image: rabbitmq:3.8.3-management
+    environment:
+        RABBITMQ_DEFAULT_USER: guest
+        RABBITMQ_DEFAULT_PASS: guest
+    ports:
+      - '5672:5672'
+      - '15672:15672'
+
+    healthcheck:
+      test: rabbitmq-diagnostics -q ping
+      interval: 30s
+      timeout: 30s
+      retries: 5
+    networks:
+      - aiida
+
+  postgres:
+    image: postgres:12
+    ports:
+      - '5432:5432'
+    networks:
+      - aiida
+    environment:
+      POSTGRES_HOST_AUTH_METHOD: trust
+    # volumes:
+    #   - db-volume:/var/lib/postgresql/data
+
+  aiida:
+    #image: "aiidateam/aiida-core:main"
+    image: "aiida-core-dev"
+    build:
```

Comment:
I think the image `aiidalab/base` from https://hub.docker.com/r/aiidalab/base/tags is what we can put here in the future if we manage to move it from aiidalab to here, correct?

Reply:
Almost. It would probably rather become `FROM` image in the `.devcontainer/Dockerfile`, since the devcontainer focuses on development (which requires e.g. some further dependencies installed).